bats/sql-commit.bats

```diff
@@ -115,4 +111,61 @@ teardown() {
     run dolt sql -q "SELECT * from dolt_commits ORDER BY Date DESC;"
     [ $status -eq 0 ]
     [[ "$output" =~ "Commit1" ]] || false
+}
+
+@test "DOLT_COMMIT immediately updates dolt log system table." {
+    run dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Commit1');
+SELECT * FROM dolt_log;
+SQL
+
+    [ $status -eq 0 ]
+    [[ "$output" =~ "Commit1" ]] || false
+}
+
+@test "DOLT_COMMIT immediately updates dolt diff system table." {
+    original_hash=$(get_head_commit)
+    run dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Commit1');
+SELECT from_commit FROM dolt_diff_test WHERE to_commit = hashof('head');
+SQL
+
+    [ $status -eq 0 ]
+    # Represents that the diff table marks a change from the recent commit.
+    [[ "$output" =~ $original_hash ]] || false
+}
+
+@test "DOLT_COMMIT updates session variables" {
+    head_variable=@@dolt_repo_$$_head
+    run dolt sql << SQL
+SELECT DOLT_COMMIT('-a', '-m', 'Commit1');
+SELECT $head_variable = HASHOF('head');
```

Comment:
This isn't really good enough... you want to verify that this variable actually reflects the true state of the world, not just that it is internally consistent within a session. You should at least verify that the variable changed before and after the commit command.